stripped/colabs/mapy_datashader_stripped.py

```diff
@@ -0,0 +1,138 @@
+"""
```

Comment:
tamtych html nie powinniśmy mieć w tym miejscu. Sugeruje je usunąć w ogóle, ale jeśli bardzo chcemy aby zostały to dać jakiś katalog `images/sample_map_outputs` czy jakoś podobnie.

Ogólnie widzę, że mamy problem z zapisywaniem notatników w odpowiednich katalogach. Zwracaj proszę uwagę na ścieżki zapisu.

Reply:
usunęłam bo faktycznie i tak te mapy tworzą się szybko, więc jest to zbędne